app/lib/features/news/model/news_post_color_data.dart

```diff
@@ -1,12 +1,52 @@
-
 import 'package:flutter/material.dart';
 
-List<Color> newsPostColors = [
-  Colors.brown,
-  Colors.red,
-  Colors.orange,
-  Colors.deepPurple,
-  Colors.purple,
-  Colors.indigo,
-  Colors.pink,
+class PostColorScheme {
+  final Color backgroundColor;
+  final Color foregroundColor;
+  final Color linkColor;
+
+  const PostColorScheme({
+    required this.backgroundColor,
+    required this.foregroundColor,
+    required this.linkColor,
+  });
+}
+
+//https://www.google.com
+final List<PostColorScheme> postColorSchemes = [
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 107, 65, 50),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 11, 72, 123),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 6, 60, 104),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 2, 106, 96),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 69, 43, 114),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 38, 48, 104),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 10, 10, 10),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
+  ),
```

Comment:
So ... I am confused. Why have a new `linkColor` and `foregroundColor` added and passed over the wire and types, if they are always the exact same? Not seeing the point of doing that at all...

Also, due to the foreground and link color being bright all these background colors are now rather dark and boring :( . I don't find this a convincing set of colors we want to work with.